src/core/sys/posix/grp.d

```diff
@@ -38,59 +38,12 @@ group* getgrnam(in char*);
 group* getgrgid(gid_t);
 */
 
-version( linux )
-{
-    struct group
-    {
-        char*   gr_name;
-        char*   gr_passwd;
-        gid_t   gr_gid;
-        char**  gr_mem;
-    }
-}
-else version( OSX )
-{
-    struct group
-    {
-        char*   gr_name;
-        char*   gr_passwd;
-        gid_t   gr_gid;
-        char**  gr_mem;
-    }
-}
-else version( FreeBSD )
-{
-    struct group
-    {
-        char*   gr_name;
-        char*   gr_passwd;
-        gid_t   gr_gid;
-        char**  gr_mem;
-    }
-}
-else version( Solaris )
-{
-    struct group
-    {
-        char*   gr_name;
-        char*   gr_passwd;
-        gid_t   gr_gid;
-        char**  gr_mem;
-    }
-}
-else version( Android )
-{
-    struct group
-    {
-        char*   gr_name;
-        char*   gr_passwd;
-        gid_t   gr_gid;
-        char**  gr_mem;
-    }
-}
-else
+struct group
 {
-    static assert(false, "Unsupported platform");
+    char*   gr_name;
+    char*   gr_passwd;
+    gid_t   gr_gid;
+    char**  gr_mem;
 }
```

Comment:
[All but gr_passwd defined in POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/grp.h.html).